eva-persistence-jdbc/src/main/kotlin/com/razz/eva/persistence/jdbc/executor/JdbcQueryExecutor.kt

```diff
@@ -18,6 +21,7 @@ import java.sql.Connection
 
 class JdbcQueryExecutor(
     private val transactionManager: TransactionManager<Connection>,
+    private val tracer: Tracer = NoopTracerFactory.create()
```

Comment:
I would appreciate it if Vertx's implementation is kept up to date as it is a first class citizen here.